File: back/mallook/src/main/java/io/ssafy/mallook/domain/member/entity/Address.java
package io.ssafy.mallook.domain.member.entity;

import jakarta.persistence.Embeddable;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Embeddable
public class Address {
    private String city; // 시
    private String district; // 구
    private String address; // 상세주소
    private String zipcode; // 우편번호
}

File: app/mallook/lib/feature/profile/model/address.dart
class Address {
  Address({
    this.city,
    this.district,
    this.address,
    this.zipcode,
  });

  Address.fromJson(dynamic json) {
    city = json['city'];
    district = json['district'];
    address = json['address'];
    zipcode = json['zipcode'];
  }

  String? city;
  String? district;
  String? address;
  String? zipcode;

  Map<String, dynamic> toJson() {
    final map = <String, dynamic>{};
    map['city'] = city;
    map['district'] = district;
    map['address'] = address;
    map['zipcode'] = zipcode;
    return map;
  }
}
